swarmrl/utils/utils.py

```diff
@@ -373,3 +374,101 @@ def create_colloids(
         direction = direction / np.linalg.norm(direction)
         cols.append(Colloid(pos=position, director=direction, type=type_, id=i))
     return cols
+
+
+def calc_ellipsoid_friction_factors_translation(
+    axial_semiaxis, equatorial_semiaxis, dynamic_viscosity
+):
+    """
+    https://link.springer.com/article/10.1007/BF02838005
+
+    Returns
+    -------
+
+    gamma_ax, gamma_eq
+        The friction coefficient for dragging the ellipsoid along its symmetry axis
+        and along one of the equatorial axes
+
+    """
+    if axial_semiaxis > equatorial_semiaxis:
+        # prolate spheroid
+        a = axial_semiaxis
+        b = equatorial_semiaxis
+        e = np.sqrt(1 - b**2 / a**2)
```

Comment:
The only part I did not find in the paper was where e comes from. Otherwise everything is implemented correctly as far as I can tell.

Reply:
Nice